README.md

```diff
@@ -10,6 +10,10 @@ Count](https://codeclimate.com/repos/58207e5ee72dfa227600001d/badges/ecd3be37c49
 
 Common notification logic for PagerDuty/Slack notifications.
 
+Please not that starting with version 0.4.0 we use the new Pager Duty events api V2, which is not backwards compatible. 
+If you don't want to change your existing API V1 calls made via this game please stick to 0.3.0. If you migrate though, 
+please make sure all the trigger_event function calls in your app got updated and meet the new requirements.
+
```

Comment:
Please update usage instructions as well.

How about starting a changelog? :)

Reply:
Typos: 
> Please **note** that (...) via this **gem**

Reply:
I would also consider addiing it to  gemspec's `post_install_message`, as additional warning (at least for some time).

Reply:
I thought about it but it'd be good to fill the gap about the previous versions, if you help me with that then sure, why not